### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Thirusan Rajadurai

### DISCUSSION THREAD

**Discussion by Thiru-sama** [heig-vd-dai-course/heig-vd-dai-course#337] [Show and tell]
https://github.com/Thiru-sama/MavenSetup

**Comment by ludelafo** [comment]
It seems I cannot access the repository. You can open it or if you want us to access it, you must add us to your project.

**Reply by ludelafo** [reply]
Thank you, everything seems good!
